### ISSUE TITLE
Convert on-order records?

### ISSUE THREAD

**Issue by dlrueda** [sul-dlss/ld4p-tracer-bullets#70]
When ordering an item, generate lightweight RDF with placeholder identifiers, without waiting for future enhancement and reconciliation, for quick publication in SearchWorks.

**Comment by darrenleeweber** [comment]
Perhaps, this would be started by placing a MARC21 binary file in a directory that is watched for changes.  When the watcher detects the file, it kicks off conversion processes that result in populating a triple store and Solr index with the new content.  The discovery system should 'see' it once it arrives in the Solr index, so long as the "placeholder identifiers" just work OK.

**Comment by dlrueda** [comment]
Yes, Darren, I think that's a good plan.  Putting files to convert into a particular directory, having that directory watched, converting and posting what's in it.

The question here is about the "on-order" nature of the record.  It will be very brief.  The question is, is it worth converting it at this stage, or do we wait for the item to arrive, with a complete bib record, before sending for conversion.  

Still need discussion! (By the LD4P-core group, not us necessarily)
